libbeat/docs/step-configure-kibana-endpoint.asciidoc

```diff
@@ -0,0 +1,20 @@
+. If you plan to use the sample Kibana dashboards provided with {beatname_uc},
+configure the Kibana endpoint:
++
+[source,yaml]
+----------------------------------------------------------------------
+setup.kibana:
+  host: "localhost:5601"
+----------------------------------------------------------------------
++
+--
+Where `host` is the hostname and port of the machine where Kibana is running,
```

Comment:
**protocol**
The default for `protocol` is `http`. However, if you specify a URL for `host`, the value of `protocol` is overridden by whatever scheme you specify in the URL